src/main/webapp/app/components/tabs/curation-data-import-tab/import-methods.tsx

```diff
@@ -108,8 +109,13 @@ export const geneCheck = async (
     if (geneData.exists()) {
       return onGeneExists();
     } else {
-      status.status = 'error';
-      status.message = 'Gene does not exist. Please create before importing';
+      if (createGene) {
+        await firebaseGeneService.createGene(hugoSymbol, isGermline);
+        return onGeneExists();
+      } else {
+        status.status = 'error';
+        status.message = 'Gene does not exist. Please create before importing';
+      }
```

Comment:
Is there a reason to not auto create gene?

Reply:
Could be typo, could be other reasons.